src/Microsoft.Azure.WebJobs.Protocols/BinderParameterDescriptor.cs

```diff
@@ -7,7 +7,9 @@ namespace Microsoft.Azure.WebJobs.Protocols
 namespace Microsoft.Azure.WebJobs.Host.Protocols
 #endif
 {
-    /// <summary>Represents a parameter bound to an IBinder.</summary>
+    /// <summary>
+    /// Represents a parameter bound to a Binder.
+    /// </summary>
     [JsonTypeName("IBinder")]
```

Comment:
Leaving the serialization protocol name as IBinder for back compat (here and below).